### ISSUE TITLE
Settings: Heading "More shortcuts" is confusing. "Where are the other shortcuts then?"

### ISSUE THREAD

**Issue by Jonta** [stefanXO/Tab-Manager-Plus#88]
```
TMP-only Firefox-profile
TMP 5.1.6
Firefox 75.0
macOS 10.11.6
```

**Comment by mastef** [comment]
I don't understand what this issue is about...

The heading is called "Other shortcuts". Below you find shortcuts to other relevant settings pages

![image](https://user-images.githubusercontent.com/762297/80933350-8a94e200-8ded-11ea-9648-37754526c0f7.png)
![image](https://user-images.githubusercontent.com/762297/80933362-97b1d100-8ded-11ea-964b-959cf4b0f4a6.png)


Then when navigating to the "Manage Extension Shortcuts" section of FF you can also adjust multiple shortcuts

![image](https://user-images.githubusercontent.com/762297/80933328-794bd580-8ded-11ea-9058-59926cb79b8e.png)

**Comment by Jonta** [comment]
For the heading "other shortcuts" to exist, I expect there to be a different "shortcuts" header first.

E.g. "Most used shortcuts" or "Shortcuts for driving your spaceship", where "other shortcuts" lists those to operate the hot chocolate dispenser. Also in the spaceship, but not the main ones people are looking for

From Wiktionary: "Not the one or ones previously referred to". And I don't see the previously referred to ones. =)

**Comment by mastef** [comment]
Fixed in 5c35424 and 5.2.0
